platform/accessibility/guidance/post-launch-audit-processes.md

```diff
@@ -0,0 +1,20 @@
+# Accessibility Post-launch Audit Processes
+Post-launch audit processes are designed to evaluate applications and static content using a wider set of devices than the [accessibility staging review](./staging-review-processes.md). The accessibility specialist(s) will also schedule your application or content for a VA 508 office quick test.
+
+You should [request a full audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=1Copenut%2C+jenstrickland&labels=508%2FAccessibility%2C+product+support&template=full-accessibility-and-508-office-audit.md&title=Full+Accessibility+%26+508+Office+Audit+%5BFeature-Name%5D) when your application is relatively code-stable in production. Ideally all issues identified during the staging review have been fixed, but your accessibility specialist will communicate known issues to the VA 508 office.
```

Comment:
this link has a lot of hardcoded details in it (assignees, labels, title...) and it isn't going to get automatically updated if we change the template. Looks like GitHub doesn't really give us a better link here.

more maintainable option might be something like:
"you should [file a "Full Accessibility & 508 Office Audit" issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose)"

Reply:
Good call. Updating.